client/data/promote-post/use-promote-post-campaigns-query.ts

```diff
@@ -59,6 +59,12 @@ export type CampaignResponse = {
 		}[];
 	};
 	is_evergreen?: boolean;
+	objective?: string;
+	objective_data: {
```

Comment:
THhs a nitpick of mine, no need to change it, but it would probably be better to just make the `objective_data` optional, and not the parameters inside them.

Like: 
```
objective_data?: {
		title: string;
		description: string;
		suitable_for_description: string;
}
```

I doubt that there would be a case where we have objective data but no title or description.